setup.py

```diff
@@ -41,12 +41,10 @@ def build_expected_version_hex(matches: Dict[str, str]) -> str:
                     serial = int(level_serial[len(level) :])
                     break
     if serial is None:
-        msg = 'Invalid PYBIND11_VERSION_PATCH: "{}"'.format(patch_level_serial)
+        msg = f'Invalid PYBIND11_VERSION_PATCH: "{patch_level_serial}"'
```

Comment:
Won't this just cause a parse error before the old python version even gets to the version check. I don't like using f-strings in setup.py for this reason.

Reply:
Our `setup.py` shouldn't run in Python <3.6, since we use python_requires. And if it does, the string error is an immediate indication that this is clearly Python 3.6+ only, as it's the most common error 3.6+ only error - I'd say it's only a hair worse than a dedicated version check & message, which we don't have (since I think they are rather pointless inside setup.py, due to python_requires). Finally, this is very much developer only, as this setup.py doesn't end up in SDists.